packages/ckeditor5-ui/tests/dropdown/dropdownview.js

```diff
@@ -469,4 +470,42 @@ describe( 'DropdownView', () => {
 			} );
 		} );
 	} );
+
+	describe( 'on dropdown close', () => {
+		it( 'should focus a dropdown button if focus is inside the dropdown', () => {
+			// Create a button inside the dropdown panel to enable focus.
+			view.panelView.children.add( new ButtonView( locale ) );
+
+			view.isOpen = true;
+
+			expect( view.panelView.element.contains( document.activeElement ) ).to.be.true;
```

Comment:
Let's be precise here. We know that `document.activeElement` is the `new ButtonView` from L477. There could be some mistakes in the code that would make this test pass by accident (what if `panelView` becomes more complex in the future and there's always something else focusable there?)